Code/src/main/java/nl/utwente/viskell/model/Component.java

```diff
@@ -0,0 +1,18 @@
+package nl.utwente.viskell.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Component extends Grouping {
```

Comment:
Possibly, but I reused some terms from the hardware domain (ports, wires, component), and want to avoid name clashes with javafx.